docs/source/installation.rst

```diff
@@ -0,0 +1,108 @@
+
+Installing FLORIS
+-----------------
+The installation procedure for FLORIS varies depending on the inteded usage.
+Developers of FLORIS will download the source code and install it in a manner
+that allows running the local source code. On the other hand, those who want
+to run the "released" version of FLORIS in their own script can install
+with a package manager.
+
+Basic Installation
+==================
+To run a released version of FLORIS, you can install via `pip` or `conda`.
+By default, this installs that latest released version. To install a specific
+version, specify that as shown below.
+
+.. code-block:: bash
+
+    # Using pip...
+    pip install floris         # Latest version
+    pip install floris==1.1.0  # Specified version number
```

Comment:
@rafmudaf what are your thoughts on changing this to 2.1.1?

Reply:
I guess in every instance of specifying a version, should we use 2.1.1 as it's the latest?

Reply:
I went with 1.1.0 because I dont want to add yet another place where we have to search for a version number when we create a new release. If specifying an old version here isn't right, I suggest using some vague:

```
pip install floris==X.Y.Z
```